src/blog/2023/08/UNS-article.md

```diff
@@ -0,0 +1,68 @@
+
+![RAMI4.0](./images/UNS/RAMI40.jpeg)
+
+However, my primary critique revolves around another issue – the structure and proposed communication methodology. Models based on layers, where each tier represents a functional area and could be covered by one or more applications, almost always lead to three fundamental problems:
+
+### Problem 1: Information Loss and Transaction Costs
+
+In the traditional model, data collection flows upward from Levels 0 to 4, while planning goes downward from Level 4 to 0. Information traversing from Level 0 to Level 4 has to pass through at least four stages. Despite theoretical lossless transmission of information, the practical scenario inevitably results in some degree of information loss between levels. The upshot is that original information from Level 0 arrives at Level 4 late, altered, or not at all.
+
+### Problem 2: The Expense of One-to-One Connections 
+
+The Automation Pyramid does not explicitly provide a data management approach. Consequently, one-to-one connections between IT systems become a necessity for data transfer between levels. For example, Level 3 IT systems need at least two connections to the adjacent levels. This can lead to thousands of one-to-one interfaces between IT systems, incurring exorbitant costs for projects and maintenance.
+
+### Problem 3: AI's Dependence on Data 
+
+Artificial intelligence (AI) requires extensive, well-organized data. Given the current architecture, data would have to be manually collected and prepared from individual systems and levels. This would invariably lead to numerous new one-to-one connections, offering no flexibility. Hence, AI and the Automation Pyramid can only collaborate in a significantly restricted manner.
+
+## IIoT Circle and Unified Namespace
+
+To overcome the limitations of traditional industrial data architecture, a paradigm shift towards a modern distributed architecture is necessary. Rather than allowing data to exist in silos within and across layers of the technology stack, data should be made accessible in a unified manner, creating a single, centralized repository. This approach facilitates a single centralized source for all enterprise systems to access the required data for their operations. This framework, which I have been calling the IIoT Circle, modernizes the original idea of the Automation Pyramid. A "Unified Namespace" operates as the core element that stores, processes, and permits data streams to be loaded and exported from other systems. All other applications communicate exclusively through the Unified Namespace, requiring only a single interface to be maintained per application. 
```

Comment:
I don't think UNS recommends having a central data store. UNS is a central representation of data but it can be stored anywhere.